Jenkinsfile-deploy

```diff
@@ -1,8 +1,48 @@
 library("tdr-jenkinslib")
+pipeline {
```

Comment:
Should we have a slack notification that this has been deployed?

Reply:
Yeah, I've added on into the success block and added a failure block as well. I've also removed the call to run the e2e tests because this lambda has no effect on them.